Source/Android/app/src/main/java/org/dolphinemu/dolphinemu/fragments/MenuFragment.java

```diff
@@ -137,6 +118,27 @@ public void onViewCreated(@NonNull View view, @Nullable Bundle savedInstanceStat
     }
   }
 
+  private void setInsets()
+  {
+    ViewCompat.setOnApplyWindowInsetsListener(mBinding.getRoot(), (v, windowInsets) ->
+    {
+      Insets cutInsets = windowInsets.getInsets(WindowInsetsCompat.Type.displayCutout());
+      int left = 0;
+      int right = 0;
+      if (ViewCompat.getLayoutDirection(mBinding.getRoot()) == ViewCompat.LAYOUT_DIRECTION_LTR)
+      {
+        left = cutInsets.left;
+      }
+      else
+      {
+        right = cutInsets.right;
+      }
+      mBinding.getRoot().setPadding(left, cutInsets.top, right,
```

Comment:
By adding padding without increasing the width of the menu, you're effectively decreasing the width of the "active" part of the menu. Please increase the width.